registry/registry.go

```diff
@@ -318,6 +302,29 @@ func (registry *Registry) ListenAndServe() error {
 	}
 }
 
+func configureDebugServer(config *configuration.Configuration) {
+	if config.HTTP.Debug.Addr != "" {
+		go func(addr string) {
+			log.Infof("debug server listening %v", addr)
```

Comment:
Or add this `import` in:

```
log "github.com/sirupsen/logrus"
```

Reply:
Should be fixed now

Reply:
there are two instances of `log.` there. Also, can we actually do the second thing I suggested - instead of calling `logrus` do the import and stick with `log.`